classes/checkout/CheckoutAddressesStep.php

```diff
@@ -261,14 +261,10 @@ public function getTemplateParameters()
 
         /** @var OrderControllerCore $controller */
         $controller = $this->context->controller;
-        if (isset($controller)) {
+        if (is_object($controller) && $controller instanceof OrderController) {
```

Comment:
I think only `instanceof` is enough?
```suggestion
        if ($controller instanceof OrderController) {
```